### ISSUE TITLE
Firefox accessibility problem with MultiSelect: focus hang

### ISSUE THREAD

**Issue by jandrew44** [palantir/blueprint#7055]
#### Environment

- __Package version(s)__:
@blueprintjs/core: 5.13.0
@blueprintjs/select: 5.2.4
- __Operating System__:
Windows 10 Home Edition
- __Browser name and version__:
Firefox 132.0.1

#### Code Sandbox

Can be reproduced on the bluerpint docs page:
https://blueprintjs.com/docs/#select/multi-select

#### Steps to reproduce

1. Go to the link mentioned above
2. Start pressing Tab key until focus is on MultiSelect input 

#### Actual behavior

As soon as focus is on MultiSelect input there is no way to make it leave it by using the Tab key.

The issue is reproducible in Firefox only, behavior is correct in Google Chrome and Microsoft Egde.

#### Expected behavior

It should be possible to switch to the next element by using the Tab key.

#### Possible solution

None.
